components/ui/Feed/FeedView.tsx

```diff
@@ -293,6 +195,8 @@ function FeedView({ feed, community = false, header }: FeedViewProps) {
   );
 }
 
+const keyExtractor = (item: PostView) => item.post.id.toString();
```

Comment:
moved this out of the component as it's a pure function, and is not stable. The alternative is to wrap it in `useCallback`, but if we don't need to it's best not to.